### ISSUE TITLE
[station_logbooks] ADIF-import via API-call

### ISSUE THREAD

**Issue by dg9vh** [magicbug/Cloudlog#1479]
When using the adif_push (https://github.com/M0LTE/adifpush) by M0LTE with station_logbooks-branch, the API does not import transfered QSOs. On Master-branch it was working.

API-keys are added with user_id and shown in API-Key-view. 

Is there any parameter added in the API-call that has to be added?

**Comment by dg9vh** [comment]
It's an issue with the adifpush - here the station_id has to be added into the call (and config and so on).

**Comment by magicbug** [comment]
Yes this isn't a bug, third party apps will need to support passing the station location id
